bindings/python/mata.pyx

```diff
@@ -851,14 +905,33 @@ cdef class Nfa:
         params = params or {"algorithm": "simulation"}
         cdef StateToStateMap state_map
         result = Nfa()
-        mata.reduce(result.thisptr, dereference(aut.thisptr), &state_map,
+        mata.reduce(result.thisptr.get(), dereference(aut.thisptr.get()), &state_map,
                     {
                         k.encode('utf-8'): v.encode('utf-8') for k, v in params.items()
                     }
         )
 
         return result, {k: v for k, v in state_map}
 
+    @classmethod
+    def reduce(cls, Nfa aut, params = None):
+        """
+        Reduce the automaton.
+
+        :param Nfa aut: Original automaton to reduce.
+        :param Dict params: Additional parameters for the reduction algorithm:
+            - "algorithm": "simulation"
+        :return: Reduced automaton
+        """
+        params = params or {"algorithm": "simulation"}
+        result = Nfa()
+        mata.reduce(result.thisptr.get(), dereference(aut.thisptr.get()), NULL,
+                    {
```

Comment:
Please, fix this ugly indentation.

Reply:
Autoformatter is trying, but sometimes it just does not work. Fixed.